src/fix/rgbfix.1

```diff
@@ -28,17 +29,26 @@
 The
 .Nm
 program changes headers of Game Boy ROM images.
-It also performs other filetype operations, such as truncation.
+It also performs other correctness operations, such as padding.
+.Pp
+Note that options can be abbreviated as long as the abbreviation is unambiguous:
+.Fl Fl verb
+is
+.Fl Fl verbose ,
+but
+.Fl Fl ver
+could also be
+.Fl Fl version .
```

Comment:
How about:

    but
    .Fl Fl ver
    is not a valid option because it could also be
    .Fl Fl version .

Reply:
Yep!